app/src/main/java/com/cornellappdev/volume/fragments/PublicationsFragment.kt

```diff
@@ -373,4 +374,19 @@ class PublicationsFragment : Fragment(), FollowingHorizontalAdapter.AdapterOnCli
     override fun onFollowClick(wasFollowed: Boolean) {
         setupPublicationsView(binding, isRefreshing = true)
     }
+
+
+    override fun onMorePublicationClicked(publication: Publication, isOnboarding: Boolean) {
+        val intent = Intent(view?.context, PublicationProfileActivity::class.java)
+        intent.putExtra(Publication.INTENT_KEY, publication)
+        intent.putParcelableExtra(
+                NavigationSource.INTENT_KEY, if (isOnboarding) {
+            NavigationSource.ONBOARDING
+        } else {
+            NavigationSource.MORE_PUBLICATIONS
+        }
+        )
```

Comment:
Indentation. There should be an Android linter somewhere, would be good to ask @aasthashah999